ios-base/Extensions/ColorExtension.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  ColorExtension.swift
+//  ios-base
+//
+//  Created by Karen Stoletniy on 12/7/21.
+//  Copyright © 2021 Rootstrap Inc. All rights reserved.
+//
+
+import UIKit
+
+// swiftlint:disable force_unwrapping
+extension UIColor {
```

Comment:
didn't we have a UIColor extension already?

Reply:
no, we didn't